docs/api/repository-signatures-resource.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Repository Signatures, NuGet API | Microsoft Docs
+author:
+- joelverhagen
+- kraigb
+ms.author:
+- joelverhagen
+- kraigb
+manager: skofman
+ms.date: 3/2/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: The repository signatures resource allows clients package sources to announce their repository signing capabilities.
+keywords: NuGet API repository signatures, nuget.org signing certificates, nuget.org package signing
+ms.reviewer:
+- karann
+- unniravindranathan
+---
+
+# Repository signatures
+
+If a package source supports adding repository signatures to published packages, it is possible for a client to
+determine the signing certificates that are be used by the package source. This resource allows clients to detect
+whether a repository signed package has been tampered or has an unexpected signing certificate.
+
+The resource used for fetching this repository signature information is the `RepositorySignatures` resource found in
+the [service index](service-index.md).
+
+## Versioning
+
+The following `@type` value is used:
+
+@type value                | Notes
+-------------------------- | -----
+RepositorySignatures/4.7.0 | The initial release
+
+## Base URL
+
+The entry point URL for the following APIs is the value of the `@id` property associated with the aforementioned
+resource `@type` value. This topic uses the placeholder URL `{@id}`.
+
+Note that unlike other resources, the `{@id}` URL is required to be served over HTTPS.
+
+## HTTP methods
+
+All URLs found in the repository signatures resource support only the HTTP methods `GET` and `HEAD`.
+
+## Repository signatures index
+
+The repository signatures index contains two pieces of information:
+
+1. Whether or not all packages found on the source are repository signed.
```

Comment:
"...are repository signed **by this repository**"?

Reply:
Fixed.